src/components/screens/PointOfInterest.js

```diff
@@ -1,24 +1,25 @@
+import _filter from 'lodash/filter';
 import PropTypes from 'prop-types';
 import React, { useContext } from 'react';
 import { StyleSheet, View } from 'react-native';
-import _filter from 'lodash/filter';
 
 import { colors, consts, device, texts } from '../../config';
+import { matomoTrackingString } from '../../helpers';
+import { useMatomoTrackScreenView } from '../../hooks';
+import { location, locationIconAnchor } from '../../icons';
+import { NetworkContext } from '../../NetworkProvider';
 import { HtmlView } from '../HtmlView';
 import { Image } from '../Image';
+import { ImagesCarousel } from '../ImagesCarousel';
+import { Logo } from '../Logo';
+import { WebViewMap } from '../map/WebViewMap';
 import { Title, TitleContainer, TitleShadow } from '../Title';
+import { TMBNotice } from '../TMB/Notice';
 import { Wrapper, WrapperWithOrientation } from '../Wrapper';
-import { PriceCard } from './PriceCard';
 import { InfoCard } from './InfoCard';
-import { OperatingCompanyInfo } from './OperatingCompanyInfo';
 import { OpeningTimesCard } from './OpeningTimesCard';
-import { ImagesCarousel } from '../ImagesCarousel';
-import { useMatomoTrackScreenView } from '../../hooks';
-import { matomoTrackingString } from '../../helpers';
-import { WebViewMap } from '../map/WebViewMap';
-import { location, locationIconAnchor } from '../../icons';
-import { NetworkContext } from '../../NetworkProvider';
-import { TMBNotice } from '../TMB/Notice';
+import { OperatingCompanyInfo } from './OperatingCompanyInfo';
+import { PriceCard } from './PriceCard';
```

Comment:
imports restructured because of eslint import plugin?
do we want that in this commit or maybe do not stash this part and fix all imports over the code base at all with one branch?

Reply:
imports restructured because i noticed they are messy. not a plugin, just vscode ;)

i did it in this file as i touched the file anyways.